deploy/scripts/src/btm/test.ts

```diff
@@ -102,21 +204,62 @@ const awsConfig = new AWSConfig({
 })
 
 const eventData = {
-  payload: __ENV.DATA_BTM_SQS_PAYLOAD
+  payloadEventsString: __ENV.DATA_BTM_SQS_PAYLOAD_EVENTS,
+  payloadTimestamp: __ENV.DATA_BTM_SQS_PAYLOAD_TIMESTAMP
```

Comment:
Might need to make a note that this will always need to be supplied at runtime under the current setup. As well as what this value should be, looks like it should be two unix timestamps (in seconds) separated by a comma?

Reply:
Yes, will update the documentation to reflect this.